### ISSUE TITLE
Why do I have to pass items in the constructor?

### ISSUE THREAD

**Issue by lectricas** [bignerdranch/expandable-recycler-view#239]
For example I want to populate my recycler after some async actions, while adapter was created earlier in onCreate method.

**Comment by paul-turner** [comment]
@lectricas We are currently limited by the fact that we can't do a general `notifyDataSetChanged()` call on the library (being tracked in #89). If you keep a reference to the list or call `getParentItemList()` you can get your original list of items, make modifications and then inform the adapter of changes using the currently implemented notify methods on the library:
```java
notifyParentItemInserted(int)
notifyParentItemRemoved(int)
notifyParentItemChanged(int)
notifyParentItemRangeInserted(int, int)
notifyChildItemInserted(int, int)
notifyChildItemRemoved(int, int)
notifyChildItemChanged(int, int)
```

**Comment by paul-turner** [comment]
Hello, we've released 3.0.0-RC1 with a fix for this issue. 

I would love your feedback with any issues you have with the solution before we make 3.0.0 final. Check out the Pull Request that contains the fix for your issue (should be referenced above), and the [changelog](https://github.com/bignerdranch/expandable-recycler-view/releases/tag/v3.0.0-RC1) for a list of changes that have come with 3.0.0 (there are a lot of them!). 

If you have any issues feel free to re-open this issue with details of what you are seeing.
